### DISCUSSION TITLE
Availability of remainder of collectors currently in Windows Exporter

### DISCUSSION THREAD

**Discussion by pcmann** [grafana-cold-storage/agent#2439] [General]
Hi,

I just want to say the work that has gone in to the Grafana Agent is brilliant!

Is there a timeline for the addition of the remainder of available collectors to be added to the windows_exporter functionality within the Grafana Agent?

Myself, selfishly would love to be able to record information around the mscluster metrics as well as scheduled_task

Also, what would be required to have custom Performance Counters to be collected?

Thanks!

**Comment by splitt3r** [comment]
I think https://github.com/grafana/agent/issues/1631 should answer your questions.

**Reply by pcmann** [reply]
Thanks! Looks like I will have to run the Windows Exporter seperately where i need additional capabilities or write to logs and pick up using the textfile capabilities

**Reply by splitt3r** [reply]
I think you can maybe disable the integrated one and scrape the manually installed one with the Grafana Agent. Anyway please upvote the issue to show interest.

**Reply by pcmann** [reply]
Brilliant, thanks - am just going to use the embedded now to test but will be following the development of the agent for updates on this :)
